dashboard/app/controllers/api/v1/pd/applications_controller.rb

```diff
@@ -4,12 +4,21 @@ class Api::V1::Pd::ApplicationsController < ::ApplicationController
   # This must be included after load_and_authorize_resource so the auth callback runs first
   include Api::CsvDownload
 
-  # GET /api/v1/pd/applications
+  # GET /api/v1/pd/applications?regional_partner=:regional_partner
   def index
+    regional_partner_value = params[:regional_partner]
```

Comment:
Is this an id? It might be worth naming the param or the variable `regional_partner_id` for clarity.

Reply:
this is not always an id, because we have special values for all regional partners' applications and for unmatched applications. I'll add a clarifying comment.